lib/msf/core/web_services/json_rpc_exception_handling.rb

```diff
@@ -0,0 +1,34 @@
+require 'msf/core/rpc'
+
+module Msf::WebServices
+  module JsonRpcExceptionHandling
+
+    def self.registered(app)
+      app.set :show_exceptions => false
+
+      app.error { |err|
+        self.request.body.rewind
```

Comment:
Just confirming: Is there the potential of body being nil?

Reply:
I looked through the docs and get be 100% certain either way, so I put in a check to be safe